shell.nix

```diff
@@ -0,0 +1,55 @@
+let
+  # sources = import ./nix/sources.nix;
+  # pkgs = import sources.nixpkgs {};
+
+  self = import ./. {};
+  inherit (self) sources pkgs;
```

Comment:
continuing from the comment left on `default.nix`...

Should I do this:

```suggestion
  sources = import ./nix/sources.nix;
  pkgs = import sources.nixpkgs {};

  self = import ./. {};
```

Reply:
Done in https://github.com/stackabletech/operator-templating/pull/323

Reply:
Personally I prefer keeping `default.nix` as the canonical source to import from.

Reply:
Cool, will mark this as resolved.